### PR TITLE
galaxy-ng 4.2.4

### PR DESCRIPTION
**Pull request by evgeni**

### CODE REVIEW

File: packages/python-galaxy-ng/python-galaxy-ng.spec [hunk 1]
```diff
@@ -41,6 +41,10 @@ Requires:       python%{python3_pkgversion}-setuptools
 # Remove bundled egg-info
 rm -rf %{pypi_name}.egg-info
 
+# drop the ansible requirement, we don't ship ansible for Python3
```
Comment (ekohl) [inline packages/python-galaxy-ng/python-galaxy-ng.spec:44]: Is there an upstream issue for this?

Comment (evgeni) [reply packages/python-galaxy-ng/python-galaxy-ng.spec:44]: no

**Review by pcreech — APPROVED** [review]
ACK from me.  Not sure if comment by @ekohl should be blocking

**Comment by evgeni** [comment]
> ACK from me. Not sure if comment by @ekohl should be blocking

I think we cleared that on IRC ;)

**Pull request merged**